backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.2
+FROM golang:1.16.2
```

Comment:
Will `go.mod` need an update too?

Reply:
You mean the line in `go.mod` saying `go 1.13` ?
I read up on that https://golang.org/ref/mod#go-mod-file-go
It seems there would be some benefit from updating it.

If you mean updating the other Go package dependencies, then I'm happy to leave that to another PR.

Reply:
:+1: